packages/common/src/types/command/PartialTargetDescriptor.types.ts

```diff
@@ -43,14 +43,31 @@ export interface RangeMark {
   excludeActive?: boolean;
 }
 
+interface PartialPosition {
+  readonly line: number;
+  readonly character: number;
+}
+
+interface PartialRange {
```

Comment:
I think I'd call this `SimpleRange` and the other one `SimplePosition`.  WDYT?

Reply:
agreed